src/App.js

```diff
@@ -20,4 +20,22 @@ export const fetchCommentsById = (article_id) => {
   return ncNewsApi.get(`/articles/${article_id}/comments`).then((response) => {
     return response.data.article
   })
+}
+
+export const incPatchVotesByArticleId = (article_id) => {
+  const patchBody = {
+    inc_votes: 1
+  }
+  return ncNewsApi.patch(`/articles/${article_id}`, patchBody).then(({data}) => {
+    return data.article
+  })
+}
+
+export const decPatchVotesByArticleId = (article_id) => {
```

Comment:
To make this more dry, couldn't you have had one function and passed it a second parameter to determine if the vote was increased or decreased?